tools/cmd/api-generator/convert_metadata.go

```diff
@@ -147,10 +148,12 @@ func extractDefaultVersion(operation *openapi3.Operation) (string, error) {
 		if err != nil {
 			return "", fmt.Errorf("unsupported version %q error: %w", mime, err)
 		}
-		defaultVersion = version
+		versions = append(versions, version)
 	}
 
-	return defaultVersion, nil
+	slices.Sort(versions)
+
+	return versions[len(versions)-1], nil
```

Comment:
Shouldn't we check that this will be inside of the bounds?

Reply:
done